src/app/mi-details/mi-details.page.html

```diff
@@ -36,22 +36,20 @@ <h1 class="text-header">{{"GENERATED_AND_EDITED_TITLE" | translate}}</h1>
 
   <div class="bottom-container">
     <p class="bottom-text">{{projectDetails.text}}</p>
-
-    <!-- @if(projectDetails.evidences.length > 0){
+    @if(projectDetails.evidences.length > 0){
       <div class="image-container">
         @for(item of projectDetails.evidences;track item){
           <div  class="image-item">
-            <div *ngIf="isFileType(item.link, 'item')">
-              <img class="image-placeholder" [src]="item.link" [alt]="item.title" />
-            </div>
-    
-            <div *ngIf="isFileType(item.link, 'pdf') || isFileType(item.link, 'doc') || isFileType(item.link, 'video')">
-              <a [href]="item.link" target="_blank" class="file-link">{{ item.title }}</a>
-            </div>
+            @if(isFileType(item.title, 'image')){
```

Comment:
wouldnt this make the page slower as this function gets called for each rendering